### ISSUE TITLE
maven-plugin should support semantic version option

### ISSUE THREAD

**Issue by petteyg** [siom79/japicmp#81]
The maven plugin doesn't accept `<parameter><semanticVersion>true</semanticVersion></parameter>`. It should support the same features as the standalone jar.

**Comment by siom79** [comment]
How do you think it should be implemented? The point is that the option for the standalone jar only outputs which position in the version to increment. Should the maven plugin just output it as part of the build process?

**Comment by petteyg** [comment]
It'd be nice to have it integrated with a release plugin to automatically increment the version (prerequisite of course being the pom using a simple nothing-but-major-minor-patch version to start with), but right now would just like to be able to output in the log and/or report the same information as the standalone.

**Comment by siom79** [comment]
I have integrated the semantic versioning information into the head of the HTML report created by the maven plugin.

If you like, you can try it on the `develop` branch:
```
git clone https://github.com/siom79/japicmp.git
cd japicmp
git checkout develop
mvn install
```

**Comment by siom79** [comment]
Released with version 0.6.1.
